### PR TITLE
Make this work with Boost 1.55.

### PR DESCRIPTION
**Pull request by igorpeshansky**

### CODE REVIEW

File: libs/network/src/server_request_parsers_impl.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 #define BOOST_SPIRIT_UNICODE
 #include <boost/spirit/include/qi.hpp>
-#include <boost/fusion/include/std_tuple.hpp>
+#include <boost/fusion/adapted/std_tuple.hpp>
```
Comment (deanberris) [inline libs/network/src/server_request_parsers_impl.cpp]: Does this still work with later Boost versions?

**Comment by deanberris** [comment]
Just curious (and I know we haven't been able to remove the Boost dependencies yet) but have you tested with later Boost releases? Also curious why you need this to work with 1.55.

File: libs/network/src/server_request_parsers_impl.cpp (hunk 1 above)
Comment (igorpeshansky) [inline libs/network/src/server_request_parsers_impl.cpp]: It should (this header is [present in 1.63](http://www.boost.org/doc/libs/1_63_0/boost/fusion/adapted/std_tuple.hpp) and on what seems to be the [development branch](https://github.com/boostorg/fusion/blob/develop/include/boost/fusion/adapted/std_tuple.hpp)).

**Comment by deanberris** [comment]
LGTM

**Comment by igorpeshansky** [comment]
I don't see why it wouldn't, but I'll test tomorrow.
As for why I need older boost versions, 1.55 is the default on older Ubuntu, and we'd much rather not package our own shared libs. Our stuff might even need to work on Wheezy and CentOS 6 (gasp!), so I may end up going even lower than 1.55.
This change, however, simply removes a gratuitous version bound, as cpp-netlib doesn't seem to be requiring any newer boost features.

**Comment by deanberris** [comment]
That's fair -- although we do use C++11 features where we can, and I know of older compilers having issues with C++11. If we're going to need to support older compilers, then a pre-C++11 build (0.11.x) would be preferred, though there's a huge delta between then and the current state of the library in 0.13-devel and master.

**Pull request merged**